internal/apiserver/server.go

```diff
@@ -476,9 +476,25 @@ func (as *apiServer) createMuxRouter(ctx context.Context, o orchestrator.Orchest
 				Methods(route.Method)
 		}
 	}
+
+	r.HandleFunc(`/api/v1/namespaces/{ns}/apis/{apiName}/api/swagger.json`, as.routeHandler(o, apiBaseURL, &oapispec.Route{
```

Comment:
Think this one needs to match:
https://github.com/kaleido-io/firefly/blob/24d075b4ec5b08601780ebac04fefc61aebbd427/internal/apiserver/server.go#L464

Reply:
It probably should, although our `routeHandler` wrapper seems to only support JSON. I can see what it would take to rewrite this without using `routeHandler` though.

Reply:
Happy for this to be deferred @awrichar 👍

Reply:
Nah I went for it